PhysicsTools/NanoAOD/plugins/BToKstllProducer.cc

```diff
@@ -1073,6 +1088,7 @@ bool BToKstllProducer::getCandX_LostTracks(unsigned int loop_index,
 					   float candX_Dz,
```

Comment:
In line 1077 of current version add the part that adds the PFCand to the candCleaner 
for both leading and subleading cases (I had myself missed it for subleading in the past)

if(isLeading || isSubleading)
checkLeptonsDuplicateTriplet[muonTrg_index][dummyPair] = loop_index;

then the lines 1078 and followings as they are
if(debug) std::cout << " passed pfcandidate subleading = " << isSubleading  << " isKloop = " << isKloop << " - pfCand - " << std::endl;
  return true;
}